docs/installation-and-operations/configuration/README.md

```diff
@@ -489,6 +489,15 @@ OPENPROJECT_2FA_SNS_SECRET__ACCESS__KEY="YOUR SECRET KEY"
 OPENPROJECT_2FA_SNS_REGION="AWS REGION"
 ```
 
+Further a REST integration could be used:
```

Comment:
This should not be documented, it is an internal API used by a previous customer

Reply:
I removed this in https://github.com/opf/openproject/commit/017e4364bb